### ISSUE TITLE
Shorthand property named "get" fails to parse

### ISSUE THREAD

**Issue by BenoitZugmeyer** [eslint/espree#100]
```js
var espree = require('espree');

espree.parse('var a = {get};', {
    ecmaFeatures: {
        objectLiteralShorthandProperties: true
    }
});
```

Raises `Error: Line 1: Unexpected token }`

Rename `get` to anything else, and it works.

**Comment by nzakas** [comment]
Ah interesting, must be triggering the getter check somehow. Languages are fun. :)

**Comment by btmills** [comment]
Working on this.
